CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Crowdin.file for crowdin integration.
```

Comment:
```suggestion
- Crowdin configuration file.
```

Reply:
I don't think you need changelog for this